$OpenBSD: patch-src_maxima_system,v 1.1 2020/05/31 07:41:13 feinerer Exp $

Upstream commit 615b4bf8b13d55a576bc60ad04f7b17d75f49021
Fix for Bug #3629, to compile with ECL 20.4.24

Index: src/maxima.system
--- src/maxima.system.orig
+++ src/maxima.system
@@ -80,8 +80,7 @@
 			  (let ((x (symbol-value (find-symbol "*AUTOCONF-LD-FLAGS*"
 							      (find-package "MAXIMA")))))
 			    (if (and x (not (string= x ""))) (list x)))
-			  :epilogue-code '(progn (require :defsystem)
-					         (cl-user::run)))))))
+			  :epilogue-code '(progn (cl-user::run)))))))
 
 (defun maxima-binary-pathname ()
   #+clisp
